portfolio/src/main/webapp/WEB-INF/ElectricityByCounty.csv

```diff
@@ -0,0 +1,59 @@
+﻿County,2018
+ALAMEDA,2951.870114
+ALPINE,9.42091
+AMADOR,137.50634
+BUTTE,722.508276
+CALAVERAS,202.779767
+COLUSA,65.824078
+CONTRA COSTA,2814.796953
+DEL NORTE,113.694574
+EL DORADO,748.960579
+FRESNO,2744.26709
+GLENN,92.525338
+HUMBOLDT,372.140643
+IMPERIAL,624.915454
+INYO,60.013788
+KERN,2381.833089
+KINGS,385.06487
+LAKE,280.210793
+LASSEN,100.540875
+LOS ANGELES,21044.97388
+MADERA,430.174209
+MARIN,649.081843
+MARIPOSA,69.125082
+MENDOCINO,289.692501
+MERCED,713.73503
+MODOC,57.040549
+MONO,88.036095
+MONTEREY,683.100426
+NAPA,360.684502
+NEVADA,447.415873
+ORANGE,7035.763296
+PLACER,1414.502822
+PLUMAS,93.456942
+RIVERSIDE,7960.740053
+SACRAMENTO,4713.027288
+SAN BENITO,124.535133
+SAN BERNARDINO,5443.731723
+SAN DIEGO,6942.482091
+SAN FRANCISCO,1462.547442
+SAN JOAQUIN,1816.39498
+SAN LUIS OBISPO,644.755402
+SAN MATEO,1510.175756
+SANTA BARBARA,777.567635
+SANTA CLARA,3860.162566

```

Comment:
I live here. This was really good idea for a dataset for creating a chart. Good job!